blog/ga-maint-for-neon.adoc

```diff
@@ -0,0 +1,227 @@
+
+image::/documentation/whatsnew/server/images/JBIDE-15189_rt_detect_5.png[]
+
+Hopefully, this will help users preemptively discover and fix errors before being hit with surprising errors when trying to use the created server adapters. 
+
+==== Support for WildFly 10.1
+
+The WildFly 10.0 Server adapter has been renamed to WildFly 10.x. It has been tested and verified to work for WildFly 10.1 installations. 
+
+=== Forge Tools
+
+==== Added Install addon from the catalog command
+
+From Forge 3.3.0.Final onwards it is now possible to query and install addons listed in the http://forge.jboss.org/addons[Forge addons page].
+
+image::/documentation/whatsnew/forge/images/4.4.1.AM3/addon-install-from-catalog.png[]
+
+==== Forge Runtime updated to 3.3.1.Final
+
+The included Forge runtime is now 3.3.1.Final. Read the official announcement http://forge.jboss.org/news/jboss-forge-3.3.1.final-is-here[here].
+
+image::/documentation/whatsnew/forge/images/4.4.1.Final/startup.png[]
+
+=== Freemarker
+
+==== Freemarker 2.3.25
+
+Freemarker library included in the Freemarker IDE was updated to latest available version 2.3.25.
+
+==== flth / fltx file extensions added
+
+The new flth and fltx extensions have been added and associated with Freemarker IDE. `flth` stands for HTML content whereas `fltx` stands for XML content.
+
+==== Overhaul of the plugin template parser
+
+The parser that FreeMarker IDE uses to extract IDE-centric information
+(needed for syntax highlighting, related tag highlighting,
+auto-completion, outline view, etc.) was overhauled. Several bugs were
+fixed, and support for the newer template language features were
+added. Also, the syntax highlighting is now more detailed inside
+expressions.
+
+==== Related tag background highlighting fixed
+
+Fixed the issue when the (by default) yellow highlighting of the
+related FTL tags shift away from under the tag as you type.
+
+==== Showing whitespace, block selection mode
+
+The standard "Show whitespace characters" and "Toggle block selection
+mode" icons are now available when editing a template.
+
+==== Improved automatic finishing of FreeMarker constructs
+
+When you type `<#`, `<@`, `${`, `\#{` and `<#--` the freemarker editor now automatically closes them.
+
+==== Error positions links on the console
+
+When a FreeMarker exception is printed to the console, the error
+position in it is a link that navigates to the error. This has worked
+long ago, but was broken for quite a while.
+
+==== Fixed auto-indentation
+
+When hitting enter, sometimes the new line haven't inherited the
+indentation of the last line.
+
+==== Updated the "database" used for auto completion
+
+Auto completion now knows all directives and "built-ins" up to
+FreeMarker 2.3.25.
+
+=== And more...
+
+You can find more noteworthy updates in on link:/documentation/whatsnew/jbosstools/4.4.1.Final.html[this page].
```

Comment:
I'm OK with taking everything since the N&N is not so big this time but let's remove "And more" section and modify the  text on lines 34. There is no more details in the N&N docs.